espnet2/gan_svs/vits/vits.py

```diff
@@ -426,6 +430,8 @@ def forward(
         feats_lengths: torch.Tensor,
         singing: torch.Tensor,
         singing_lengths: torch.Tensor,
+        asr_feats: torch.Tensor = None,
```

Comment:
asr feats is confusing. Please consider changing a name for this one.

Reply:
> asr feats is confusing. Please consider changing a name for this one.

How about ssl_feats?